src/app/cluster/details/cluster/node-list/template.html

```diff
@@ -88,7 +88,7 @@
           <ng-container *ngIf="element.status.addresses && getAddresses(element) as addresses">
             <div fxLayout="column"
                  fxLayoutGap="5px"
-                 [ngClass]="{'node-ip-addresses-spacing': (addresses.externalIP && addresses.internalIPs.length) || addresses.internalIPs.length > 1}">
+                 [ngClass]="{'node-ip-addresses-spacing': (addresses.externalIPs.length && addresses.internalIPs.length) || addresses.internalIPs.length > 1 || addresses.externalIPs.length > 1}">
```

Comment:
Shouldn't this check be simple
In both cases it will be added.
addresses.externalIPs.length && addresses.internalIPs.length -> 1, 1 => true => class added
OR
addresses.externalIPs.length || addresses.internalIPs.length  -> 0, 1 -> true => class added

IMO, just adding -> `addresses.externalIPs.length  || addresses.internalIPs.length` will work

to simple => All scenarios can be covered by Above check
- 0,0 => false 
- 0,1 => true
- > 1,1 or > 1,> 1 => true
thoughts?

Reply:
We don't want to add this class in cases 0,1 or 1,0. We only want this when there are at least 2 entries in the column i.e. 1 Int. IP, 1 Ext. IP or 2 Int. IPs, 0 Ext. IP or 2 Ext.IPs, 0 Int. IP

Reply:
got it! thanks